multi-agent/bootstrap/container.py

```
@@ -312,6 +319,28 @@ def __init__(self, cfg: AppConfig):
 
         self._initialized = True
 
+    @staticmethod
+    def _create_overrides_store():
```

Ruff (0.15.20)

[warning] Missing return type annotation for staticmethod `_create_overrides_store`

(ANN205)